packages/runtime-core/src/renderer.ts

```diff
@@ -2230,7 +2231,11 @@ function baseCreateRenderer(
       unregisterHMR(instance)
     }
 
-    const { bum, scope, update, subTree, um } = instance
+    const { bum, scope, update, subTree, um, m, u } = instance
+
+    // #9264 invalidate queued lifecycle hooks
```

Comment:
I am wondering if we should invalidate the `bu`, `bm` as well ?
I am not familiar with code base yet, but is it considerable to invalidate the scheduled jobs based on the `job.ownerInstance.uid` ?

Reply:
> I am wondering if we should invalidate the bu, bm as well ?

Because they are called synchronously, not put in the queue.

> I am not familiar with code base yet, but is it considerable to invalidate the scheduled jobs based on the job.ownerInstance.uid ?

`ownerInstance` is a dev-only property.